qbft/spectest/tests/controller/decided/current_instance.go

```diff
@@ -32,8 +34,52 @@ func CurrentInstance() tests.SpecTest {
 					DecidedCnt: 1,
 					DecidedVal: testingutils.TestingQBFTFullData,
 				},
-				ControllerPostRoot: "9a5519c1584d326e7c20cb1a824af5dc7139bac2f4d1ced2ff13b5aacfe9ab33",
+				ControllerPostRoot: currentInstanceStateComparison().Register().Root(),
 			},
 		},
 	}
 }
+
+func currentInstanceStateComparison() *qbftcomparable.StateComparison {
```

Comment:
What do you think about separating to a file called current_instance_sc.go? For readability

Reply:
Separating it into a separate file will cause a proliferation in the explorer's view, and will make it hard to know what tests we have, and how many of them.

Since we put the `Comparison()` function at the bottom of the file for each test, it doesn't hurt readability too much.
If you still insist we will move it.